### DISCUSSION TITLE
Completion trigger characters.

### DISCUSSION THREAD

**Discussion by MechMel** [eclipse-langium/langium#931] [Q&A] [answered]
Is there a way to override the completion trigger characters?

**Comment by msujew** [comment, marked as answer]
You can override the [`DefaultLanguageServer`](https://github.com/langium/langium/blob/9493cdb10de0c9a79195485d5bd0208d20b55ec6/packages/langium/src/default-module.ts#L130) class' `buildInitializeResult` method with something like this:

```ts
protected override buildInitializeResult(params: InitializeParams): InitializeResult {
  const result = super.buildInitializeResults(params);
  result.capabilities.completion = { ... };
  return result;
}
```

**Reply by MechMel** [reply]
Sweet. Thanks!
